docs/modules/ROOT/assets/images/fire_product_registration.md

```diff
@@ -0,0 +1,30 @@
+# Fire product registration
+
+```mermaid
+sequenceDiagram
+    actor PO as ProductOwner
+    actor IO as InstanceOperator
+    participant G as GIF
+    participant P as FireProduct
+    participant F as PolicyFlow
+    participant R as FireRiskpool
+    participant T as ERC20Token
+    
+    IO ->> G: grant role to ProductOwner
+    PO ->>+ P: deploy FireProduct contract
```

Comment:
Since deployment can happen independently before the rest of the actions, i would put this as the first interaction.